src/components/website-analyzer/EnhancedScheduledScans.tsx

```
@@ -0,0 +1,736 @@
+    if (diffHours > 0) return `in ${diffHours} hour${diffHours > 1 ? 's' : ''}`;
+    if (diffMins > 0) return `in ${diffMins} minute${diffMins > 1 ? 's' : ''}`;
+    if (diffMins < 0) return 'overdue';
+    return 'soon';
+  };
+
+  const getFrequencyLabel = (frequency: string, dayOfWeek?: number, dayOfMonth?: number) => {
+    switch (frequency) {
+      case 'daily':
+        return 'Daily';
+      case 'weekly':
+        const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)